src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java

```diff
@@ -0,0 +1,71 @@
+package ru.yandex.practicum.filmorate.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+import ru.yandex.practicum.filmorate.exceptions.ValidationException;
+import ru.yandex.practicum.filmorate.model.Film;
+import ru.yandex.practicum.filmorate.model.IdGenerator;
+
+import java.time.LocalDate;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+public class FilmController {
+    private static final Logger log = LoggerFactory.getLogger(FilmController.class);
+    private final Map<Integer, Film> films = new HashMap<>();
+    private static final int MAX_DESCRIPTION_LENGTH = 200;
+    private static final LocalDate FIRST_CINEMA_DATE = LocalDate.of(1895, 12, 28);
+
+    @GetMapping("/films")
+    public Collection<Film> findAll() {
+        log.debug("Текущее количество фильмов: {}", films.size());
+        return films.values();
+    }
+
+    @PostMapping("/films")
+    public Film create(@RequestBody Film film) throws ValidationException {
+        film.setId(IdGenerator.generateFilmId());
+        if (filmCheck(film)) {
```

Comment:
лучше пусть будет метод
```suggestion
        validate(film)
        //other code
```
который либо бросает исключение, либо ничего не делает и не возвращает
тогда не нужны будут if else